SPDX-License-Identifier: LGPL-2.1-or-later

Do not use a ^| table marker that OpenBSD groff rejects.

Index: doc/man/man3/XmDisplay.3
--- doc/man/man3/XmDisplay.3.orig	Fri Jan  4 22:55:51 2002
+++ doc/man/man3/XmDisplay.3	Fri Apr 23 20:39:33 2010
@@ -263,7 +263,7 @@ decision matrix is described in the following table\&.
 .TS
 tab() box;
 c| c s s s c
-^| l| l| l| l l.
+l| l| l| l| l l.
 \fBDrag Initiator
 Protocol Style\fP\fBDrag Receiver Protocol Style\fP
 ______
